doc/validators-example.txt

```diff
@@ -23,26 +21,36 @@
 #
 #   Examples:
 #    n9KorY8QtTdRx7TVDpwnG9NvyxsDwHUKUEeDLY3AkiGncVaSXZi5
-#    n9MqiExBcoG19UXwoLjBJnhsxEhAZMuWwJDRdkyDz1EkEkwzQTNt John Doe
+#    n9MqiExBcoG19UXwoLjBJnhsxEhAZMuWwJDRdkyDz1EkEkwzQTNt
 #
+# [validator_list_sites]
 #
+#   List of URIs serving lists of recommended validators. Validator lists are
+#   fetched using https if a scheme is not specified.
```

Comment:
Scheme is now required
https://github.com/ripple/rippled/pull/1842/commits/b33b698392821fef0453b98f31deed7051aee44b